docs/semgrep-code/semgrep-pro-engine-intro.md

```diff
@@ -136,7 +136,7 @@ To enable Semgrep Pro Engine in the Semgrep Cloud Platform, follow these steps:
 
 1. Sign in to [Semgrep Cloud Platform](https://semgrep.dev/login).
 1. Select **[Settings](https://semgrep.dev/orgs/-/settings)**.
-1. Enable the <i class="fa-solid fa-toggle-large-on"></i> **Semgrep Pro Engine beta** toggle.
+1. Ensure that the <i class="fa-solid fa-toggle-large-on"></i> **Pro Engine beta** toggle is enabled.
```

Comment:
```suggestion
1. Ensure that the <i class="fa-solid fa-toggle-large-on"></i> **Pro Engine** toggle is enabled.
```

Reply:
@armchairlinguist I still see "beta" in Playground:
<img width="444" alt="Screenshot 2023-06-20 at 19 35 00" src="https://github.com/returntocorp/semgrep-docs/assets/26669307/988136b1-68a3-46a0-968f-43e461962078">

Reply:
Yep - turns out it should be the other way around (beta everywhere), my bad.